pkg/utils/logger/logger.go

```diff
@@ -22,8 +22,8 @@ import (
 )
 
 const (
-	envLogLevel    = "ECS_CNI_LOGLEVEL"
-	envLogFilePath = "ECS_CNI_LOG_FILE"
+	envLogLevel    = "EKS_CNI_LOGLEVEL"
```

Comment:
this should definitely not have `EKS` in the name. A suggestion here would be `AWS_VPC_CNI_K8S_LOGLEVEL`

Reply:
Done